src/Blueprint/Blueprint/Cmdlets/BlueprintDefinition/ImportAzureRMBlueprint.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.Azure.Commands.Blueprint.Common;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using System.Management.Automation;
+using static Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet("Import", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "BlueprintWithArtifact", SupportsShouldProcess = true, DefaultParameterSetName = ParameterSetNames.ImportBlueprintParameterSet), OutputType(typeof(string))]
```

Comment:
@filizt what does this cmdlet do? It says it returns a string, but I don't see anything being written to the output stream

Reply:
It imports the blueprint with its artifacts (meaning it creates a new blueprint with its artifacts with in a MG or a subscription). There is no output produced from this operation. I'll remove the `OutputType(typeof(string))` from the cmdlet.

Reply:
@filizt same comment was the `Export-AzBlueprint` cmdlet -- you will want to follow the `-PassThru` pattern here